### PR TITLE
Sticky Navbar 

### PR DESCRIPTION
**Pull request by fmasood23**

### CODE REVIEW

File: src/Components/Navbar/UserNavbar.js [hunk 1]
```diff
@@ -81,12 +82,14 @@ export default function UserNavBar(props) {
...
 
+
   useEffect(() => {
     // eslint-disable-next-line
   }, [])
 
   return (
-    <div className='user-nav'>
+
+    <div className={'user-nav'}>
```
Comment (IsitaB) [inline src/Components/Navbar/UserNavbar.js:92]: get rid of the curly braces

**Review by IsitaB — CHANGES_REQUESTED** [review]
The navbar overlaps each page's first component. otw it works great :)

why do we need `margin-top:50px` in all these files? is there a better way to handle this?

File: src/Components/Navbar/UserNavbar.js [hunk 2]
```diff
@@ -81,12 +81,14 @@ export default function UserNavBar(props) {
     setMenuIsOpen(!menuIsOpen);
   };
 
-  useEffect(() => {
+
+  useEffect(() =>
```
Comment (IsitaB) [inline src/Components/Navbar/UserNavbar.js:85]: why do we need this? get rid of it

**Review by IsitaB — CHANGES_REQUESTED** [review]
get rid of the hardcoded margin-tops

make the navbar's position:sticky instead

**Review by IsitaB — APPROVED** [review]

**Review by jerrylee17 — APPROVED** [review]

**Review by IsitaB — CHANGES_REQUESTED** [review]
delete the empty useEffect block and take package-lock out of the PR since you havent made any dependency changes

**Review by IsitaB — APPROVED** [review]

**Pull request merged**